intelmq/bots/parsers/shadowserver/_config.py

```diff
@@ -223,1375 +223,1407 @@ def force_base64(value: Optional[str]) -> Optional[str]:
+        ('extra.', 'public_source', validate_to_none),
+        ('extra.', 'application', validate_to_none),
         ('extra.', 'version', validate_to_none),
-        ('source.asn', 'asn', invalidate_zero),
+        ('extra.', 'event_id', validate_to_none),
+        ('destination.url', 'http_url', convert_http_host_and_url, True),
+        ('destination.fqdn', 'http_host', validate_fqdn),
+        ('extra.', 'http_referer', validate_to_none),
+    ],
+    'constant_fields': {
+        'classification.taxonomy': 'other',
+        'classification.type': 'other',
+        'classification.identifier': 'sinkhole-http-referer',
+    },
+}
+
+malware_url = {
+    'required_fields': [
+        ('time.source', 'timestamp', add_UTC_to_timestamp),
+        ('source.ip', 'ip', validate_ip),
+    ],
+    'optional_fields': [
+        ('extra.', 'url', validate_to_none),
+        ('extra.', 'host', validate_to_none),
+        ('source.asn', 'asn', convert_int),
         ('source.geolocation.cc', 'geo'),
         ('source.geolocation.region', 'region'),
         ('source.geolocation.city', 'city'),
-        ('extra.', 'naics', invalidate_zero),
-        ('extra.', 'sic', invalidate_zero),
-        ('source.local_hostname', 'server_name'),
-        ('extra.', 'instance_name', validate_to_none),
-        ('extra.', 'tcp_port', convert_int),
-        ('extra.', 'named_pipe', validate_to_none),
-        ('extra.', 'response_length', convert_int),
-        ('extra.', 'amplification', convert_float),
-        ('extra.', 'sector', validate_to_none),
+        ('extra.source.naics', 'naics', invalidate_zero),
+        ('extra.source.sector', 'sector', validate_to_none),
+        ('malware.name', 'tag'),
+        ('extra.', 'source', validate_to_none),
+        ('malware.hash.sha256', 'sha256', validate_to_none),
+        ('extra.', 'application', validate_to_none),
     ],
     'constant_fields': {
-        'classification.taxonomy': 'vulnerable',
-        'classification.type': 'vulnerable-system',
-        'classification.identifier': 'open-mssql',
-        'protocol.application': 'mssql',
+        'classification.type': 'malware-distribution',
+        'classification.taxonomy': 'malicious-code',
+        'classification.identifier': 'malware-url',
     },
 }
 
-# https://www.shadowserver.org/wiki/pmwiki.php/Services/Open-MongoDB
-open_mongodb = {
+phish_url = {
     'required_fields': [
         ('time.source', 'timestamp', add_UTC_to_timestamp),
-        ('source.ip', 'ip'),
-        ('source.port', 'port'),
+        ('source.ip', 'ip', validate_ip),
     ],
     'optional_fields': [
-        ('protocol.transport', 'protocol'),
-        ('source.reverse_dns', 'hostname'),
-        # ('classification.identifier', 'tag'),  # always set to 'open-mongodb' in constant_fields
-        ('extra.', 'version', validate_to_none),
-        ('source.asn', 'asn', invalidate_zero),
+        ('extra.', 'url', validate_to_none),
+        ('extra.', 'host', validate_to_none),
```

Comment:
If that's the URL/host of the phishing page:
```suggestion
        ('source.url', 'url', validate_to_none),
        ('source.fqdn', 'host', validate_to_none),
```

Reply:
Updated:

```
-        ('extra.', 'url', validate_to_none),
-        ('extra.', 'host', validate_to_none),
+        ('source.url', 'url', convert_http_host_and_url, True),
+        ('source.fqdn', 'host', validate_fqdn),
```